### PR TITLE
Creating option for Communication direction

### PR DESCRIPTION
**Pull request by csark**
Hoping to add to the next release the ability to choose the communication direction when installing a standalone scan engine. Enables users that choose this option to connect to new engines quicker when environment is set up where the console talk to engines instead of the other way around

### CODE REVIEW

File: attributes/default.rb [hunk 1]
```diff
@@ -47,6 +47,8 @@
 default['nexpose']['company_name'] = 'Rapid7'
 # Install type (typical || engine)
 default['nexpose']['component_type'] = 'typical'
+# Communication direction (1 => Engine->Console || 2 => Console->Engine)
```
Comment (bcall-r7) [inline attributes/default.rb]: So just a correction. 0 is Engine to console, 1 is Console to engine.

**Comment by bcall-r7** [comment]
I going to merge this and fix this comment after the merge so I can include this in the release today. Just wanted to give you a heads up about the values.

**Comment by csark** [comment]
Oh I was just following the numbering on the installer. But whatever works
for you!

**Comment by bcall-r7** [comment]
PR to update the comment: https://github.com/rapid7-cookbooks/nexpose/pull/43. Thanks for the contribution :D

**Pull request merged**